source/main.html

```diff
@@ -1,10 +1,52 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <link rel="stylesheet" href="main.css" />
     <meta chartset="UTF-8"/>
-    <title>Home page</title>
+    <title>My Journal</title>
 
     <!-- Main Stylesheet & Scripts -->
 </head>
-
+<body>
+        <header>
+            <div class="appinfo">
+                <img src="" alt="logo">
+                <h1 class="app_title">MyJournal</h1>
+            </div>
+            <div class="user_info">    <h2 class="user_name">User Name</h2>
+                <input type="button" class="sign_out" value="Sign Out">
+            </div>
+        </header>
+        <main>
+        <div class="left_create">
+            <button class="create" value="Create">
+                <img src="" alt="create_image">
+                Create
+            </button>
+        </div>
+        <div class="filters">
+            <img src="" alt="filter_image">
+            <h2>Filter by</h2>
+            <h3>Label</h3>
+            <div class="labels">
+                <input type="button" class="filterby_label1" value="Hapiness">
+                <input type="button" class="filterby_label2" value="Sadness">
+                <input type="button" class="filterby_label3" value="Anger">
+                <input type="button" class="filterby_label4" value="Fear">
+                <input type="button" class="filterby_label5" value="Surprise">
+            </div>
```

Comment:
label buttons might be better served by a checkboxes to allow filtering to several different labels.

Also for styling, I think it would be better if they all shared one class, like label_filter_checkbox and then have individual ids like filterby_lablel1 etc.

Reply:
Sure I'll make the change